bin/prettier.js

```diff
@@ -264,6 +266,11 @@ if (stdin) {
         if (output) {
           console.log(output);
         }
+      } else if (argv["list-different"]) {
+        if (input !== output) {
+          console.log(filename);
```

Comment:
I'm not sure if I should print it on stdout or stderr. @llimllib could you quickly check what gofmt does?